.github/workflows/maven.yml

```diff
@@ -68,4 +68,4 @@ jobs:
         run: ./mvnw -e -B -V -ntp javadoc:javadoc
 
       - name: Build site
-        run: ./mvnw -e -B -V -ntp site
+        run: ./mvnw -e -B -V -ntp site site:stage
```

Comment:
That's exactly what I meant - instead of "mvn site site:stage" shouldn't "mvn site:stage" be enough?

Is the stage build correct as the pom does not contain a valid site URL:
  <distributionManagement>
    <site>
      <id>staging</id>
      <url>file:bogus</url>
    </site>
  </distributionManagement>

Therefore it was not used before.